data_steward/tools/historical_retraction.py

```diff
@@ -0,0 +1,90 @@
+# coding=utf-8
+"""
+Simulate cron retraction
+"""
+import logging
+
+import bq_utils
+import constants.global_variables
+from cdr_cleaner import clean_cdr
+from cdr_cleaner.args_parser import add_kwargs_to_args
+from constants.cdr_cleaner.clean_cdr import CRON_RETRACTION
+from gcloud.bq import BigQueryClient
+from retraction import retract_data_bq, retract_data_gcs
+
+
+def run_cron_retraction():
+    """
+    Simulate cron retraction
+
+    :return:
+    """
+    constants.global_variables.DISABLE_SANDBOX = True
+    project_id = bq_utils.app_identity.get_application_id()
+    hpo_id = bq_utils.get_retraction_hpo_id()
+    retraction_type = bq_utils.get_retraction_type()
+    pid_table_id = bq_utils.get_retraction_pid_table_id()
+    sandbox_dataset_id = bq_utils.get_retraction_sandbox_dataset_id()
+
+    # Dataset and table containing list of datasets
+    datasets_to_retract_dataset = bq_utils.get_retraction_dataset_ids_dataset()
+    datasets_to_retract_table = bq_utils.get_retraction_dataset_ids_table()
+
+    # retract from bq
+    if not datasets_to_retract_table or not datasets_to_retract_dataset:
+        logging.info(
+            f"Retraction cannot run without RETRACTION_DATASET_IDS_TABLE and RETRACTION_DATASET_IDS_DATASET"
+        )
+        return 'retraction-skipped'
+
+    bq_client = BigQueryClient(project_id)
+    dataset_query_job = bq_client.query(
+        f"SELECT * FROM {project_id}.{datasets_to_retract_dataset}.{datasets_to_retract_table}"
+    )
+    dataset_ids_result = dataset_query_job.result()
+    dataset_ids = dataset_ids_result.to_dataframe()["datasets"].to_list()
+    logging.info(f"Dataset id/s to target from table: {dataset_ids}")
+    logging.info(f"Running retraction on BQ datasets")
+
+    # retract from default dataset
+    # retract_data_bq.run_bq_retraction(project_id,
```

Comment:
Do you think we should uncomment this part?

Reply:
You are right, I missed this. Let me uncomment it.